src/SharedModule/Components/MasterLayout/MasterLayout.tsx

```diff
@@ -1,13 +1,16 @@
-import React from 'react'
 import Navbar from '../../../UsersModule/Components/Navbar/Navbar'
 import Footer from '../../../UsersModule/Components/Footer/Footer'
 import { Outlet } from 'react-router-dom'
+import { Box } from '@mui/material'
 
 export default function MasterLayout() {
   return (
     <div>
         <Navbar/>
+        <Box sx={{height:"100vh"}}>
```

Comment:
We can change this to minHeight:"100dvh"
min height, because setting a fixed height will cause elements beyond this height to overflow.